search-sentiments-map/routes/sentiment.js

```diff
@@ -1,53 +1,54 @@
 /** Server-side script that uses the Google Cloud Natural Language API to get the sentiment score. */
 const express = require('express');
-var router = express.Router();  // Use a Router to divide the app into modules.
+var router = express.Router();  // Using Router to divide the app into modules.
+var allSettled = require('promise.allsettled');
 const language = require('@google-cloud/language');
 
-var bodyParser = require('body-parser');  
-// Create application/x-www-form-urlencoded parser.
-var urlencodedParser = bodyParser.urlencoded({ extended: false })
-
-/** Returns the sentiment score of inputted search results. */  
-router.post('/', urlencodedParser, (req, res) => { 
-  const searchTopic = req.body.searchTopic;
-  console.log('ntarn debug: '+ searchTopic);
-  getSentimentScore(searchTopic).then(score => {
-    // Prepare output in JSON format.  
-    response = {  
-      sentimentScore:score,  
-    };  
-    // TODO(ntarn): Remove console.log statement when finished with feature. 
-    console.log('ntarn debug: searchTopic score' + response.sentimentScore);  
+var bodyParser = require('body-parser');
+var textParser = bodyParser.text();
+
+router.post('/', textParser, (req, res) => {
+  getSentimentScore(req.body).then((sentimentScore) => {
+    response = {
+      score: sentimentScore,
+    };
+    // TODO(ntarn): Remove comment when done with feature.
+    console.log('ntarn debug: score' + response.score);
     res.end(JSON.stringify(response));
   }).catch(err => {
     console.log(err);
-  })
+  });
 });
 
 /** 
- * Analyzes the sentiment score for the search topic.
- * @param {string} searchTopic Search topic to get data for.
- * @return {double} Sentiment score for the search topic.
+ *  Gets the sentiment score of an inputted search result title and snippet.
+ *  @param {string} searchResultTitleSnippet The search result combined title and snippet.
```

Comment:
"search result combined title and snippet" rephrase? You don't have to limit the number of words you use :)